.circleci/config.yml

```diff
@@ -69,6 +69,6 @@ workflows:
   version: 2
   build_and_test:
     jobs:
-      - build__CONDA_PY_27
+      - build__CONDA_PY_34
```

Comment:
conda-forge does not build for Python 3.4 due to CI resource constraints, sorry.

PS: the feedstock boilerplate should not me changed "by hand," we use conda-smithy to re-render the feedstock and revert this change.